src/notation/internal/notationconfiguration.cpp

```diff
@@ -89,6 +89,8 @@ static const Settings::Key NEED_TO_SHOW_ADD_GUITAR_BEND_ERROR_MESSAGE_KEY(module
 
 static const Settings::Key PIANO_KEYBOARD_NUMBER_OF_KEYS(module_name,  "pianoKeyboard/numberOfKeys");
 
+static const Settings::Key PIANO_KEYBOARD_NOTATED_PITCH(module_name,  "pianoKeyboard/notatedPitch");
```

Comment:
Here too:
```suggestion
static const Settings::Key PIANO_KEYBOARD_USE_NOTATED_PITCH(module_name,  "pianoKeyboard/useNotatedPitch");
```
(and in other places)